### PR TITLE
UIIN-1023 Instance. Remove The New button

### PR DESCRIPTION
**Pull request by NikitaSedyx**
## Purpose

https://issues.folio.org/browse/UIIN-1023

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [x] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [x] Code coverage on new code is 80% or greater
  - [x] Duplications on new code is 3% or less
  - [x] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [x] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/components/InstancesList/InstancesList.js [hunk 1]
```diff
@@ -225,6 +232,21 @@ class InstancesView extends React.Component {
...
     return (
       <Fragment>
+        <IfPermission perm="ui-inventory.instance.create">
+          <Button
+            buttonStyle="dropdownItem"
+            id="clickable-newinventory"
+            onClick={buildOnClickHandler(this.openCreateInstance)}
+          >
+            <Icon
+              icon="plus-sign"
```
Comment (aliaksei-chumakou) [inline src/components/InstancesList/InstancesList.js:242]: IMHO, icon could be removed, it contradicts folio style, check bad example https://ux.folio.org/docs/guidelines/components/button/

Comment (NikitaSedyx) [reply src/components/InstancesList/InstancesList.js:242]: it's requirement

Comment (NikitaSedyx) [reply src/components/InstancesList/InstancesList.js:242]: and it's dropdown item

Comment (aliaksei-chumakou) [reply src/components/InstancesList/InstancesList.js:242]: so it's weird that requirement doesn't follow style guidelines or vice versa

Comment (NikitaSedyx) [reply src/components/InstancesList/InstancesList.js:242]: style guide is about single button and not about dropdown menu

**Comment by zburke** [comment]
FYI `data-test-*` attributes may only be used in unit/functional tests, i.e. BigTest. They are stripped from the production build and therefore cannot be used in any nightmare/integration tests, i.e. anything within `/test/ui-testing`. This goes for all repositories.

**Pull request merged**